packages/demo/src/styles.scss

```
@@ -153,3 +153,10 @@ code {
     }
   }
 }
+
+// Take this, you inflexible header custom content slots
+sp-intranet-header .settings:has(.version-switch) {
+  > div > .p-3 {
+    padding: 0 !important;
```

```suggestion
    .version-switch { margin: -1rem; }
```
`:has()` is not yet supported by Firefox

---

Done, removed the global styles.